### ISSUE TITLE
Need to add R 4.0, 4.1 (and 4.2?) to CI tests

### ISSUE THREAD

**Issue by agvandervelde** [Paradigm4/SciDBR#256]
It looks like right now we're testing only on R versions < 4.0, whereas many installs now use 4.0, 4.1, 4.2.

**Comment by ksens** [comment]
Good catch @agvandervelde 

I started #257 . Let's wait till folks get back to get a consensus on the following questions:

Q1. Would folks be OK dropping R 3.5 from the test-matrix? 
Q2. Would folks be OK dropping ubuntu-xenial from the test-matrix? 

As reference, here is the current test-matrix for scidbR:

<img width="653" alt="image" src="https://user-images.githubusercontent.com/13973052/209156836-c482cd56-1588-4f3f-9862-c62106450b45.png">

**Comment by agvandervelde** [comment]
I think it's probably a good idea to keep Ubuntu, or switch it out with Debian. It would give us more confidence that it works well across the board and that we haven't overfitted it to CentOS 7 or other redhat-derived distributions for that matter.

**Comment by ksens** [comment]
@rvernica 

Thanks for the inputs.

Once #257 is merged in, we can target another PR that refreshes
- the scidb docker image
- the host OS used for scidbR testing
... but we can work on that as a separate PR.
